meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_0.1.1.bb

```diff
@@ -2,6 +2,7 @@ LICENSE = "CLOSED"
 LICENSE_FLAGS = "commercial"
 
 SUB_FOLDER_arm = "armhf"
+SUB_FOLDER_aarch64 = "arm64"
```

Comment:
We should also add it in COMPATIBLE_HOSTS

Reply:
Indeed we should! Fixed.